gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/client/impl/ClientManagerImpl.java

```diff
@@ -97,6 +97,7 @@ public void onEvent(Event<ApplicationEvent, Payload> event) {
                     removeClient(event.content().getId());
                     gatewayMetricProvider.decrementApp();
                     break;
+                default: break;
```

Comment:
break in default?

Can we not use a new switch here?

Reply:
https://sonarcloud.io/organizations/gravitee-io/rules?open=java%3AS131&rule_key=java%3AS131